docs/docs.js

```diff
@@ -298,6 +322,47 @@ const Example = class extends React.Component {
             />
           </div>
         </div>
+        <hr className='margin-y-50' />
+        <div className='ClockDemo--wrapper'>
+          <p>
+            Same clock, but big arm freezes at some point displaying use of disable property
+          </p>
+          <div className='ClockDemo'>
+            <Plx
+              className='ClockDemo-arm'
+              disabled={ window.pageYOffset > 1600 }
+              parallaxData={ [
+                {
+                  start: 0,
+                  duration: 2000,
+                  properties: [
+                    {
+                      startValue: 0,
+                      endValue: 1440,
+                      property: 'rotate',
+                    },
+                  ],
+                },
+              ] }
+            />
+            <Plx
+              className='ClockDemo-arm ClockArm--small'
+              parallaxData={ [
```

Comment:
It makes sense to add same `disabled` prop here as well.

Reply:
Sure, no problem